login/logout.js

```diff
@@ -1,7 +1,9 @@
 const core = require('@actions/core')
 const exec = require('@actions/exec');
+const MetritonClient = require('../src/metrics')
 
 const telepresenceLogout = async function(){
+    MetritonClient.sendMetricsReport('logout')
```

Comment:
I'd push this line after the check if the session is created because if the `telepresence_session_created` is not set it won't execute `telepresence logout`